dipy/viz/horizon/tab/tests/test_peak.py

```diff
@@ -0,0 +1,230 @@
+from unittest.mock import patch
+
+import numpy as np
+import pytest
+
+from dipy.direction.peaks import PeaksAndMetrics
+from dipy.testing.decorators import set_random_number_generator, use_xvfb
+from dipy.utils.optpkg import optional_package
+from dipy.viz.horizon.tab.base import HorizonTab, HorizonUIElement
+from dipy.viz.horizon.tab.peak import PeaksTab
+from dipy.viz.horizon.visualizer.peak import PeaksVisualizer
+
+fury, has_fury, setup_module = optional_package("fury", min_version="0.10.0")
+skip_it = use_xvfb == "skip"
```

Comment:
`skip_it = use_xvfb == "skip" or not has_fury`

then you can use `skip_it` everywhere